src/frontend/staff-portal/src/app/services/dispute.service.ts

```diff
@@ -40,18 +43,12 @@ export class DisputeService implements IDisputeService {
      *
      * @param none
      */
-    public getDisputesWithPendingUpdates(): Observable<Dispute[]> {
+    public getDisputesWithPendingUpdates(): Observable<DisputeWithUpdates[]> {
       return this.disputeApiService.apiDisputeDisputeswithupdaterequestsGet()
         .pipe(
-          map((response: Dispute[]) => {
+          map((response: DisputeWithUpdates[]) => {
             this.logger.info('DisputeService::getDisputesWithPendingUpdates', response);
-            this._disputes.next(response);
-            response.forEach(dispute => {
-              dispute = this.joinGivenNames(dispute);
-              dispute = this.joinLawyerNames(dispute);
-              dispute = this.joinAddressLines(dispute);
-            });
-
+            if (response.length > 1) this._disputesWithUpdates.next(response);
```

Comment:
I think even response.length equals 0, you still need to update the BehaviorSubject